fastlane/Fastfile

```diff
@@ -166,18 +166,33 @@ def push_pods
   pod_push(path: "PurchasesHybridCommon.podspec", synchronous: true)
 end
 
-def carthage_archive
+def archive
+  framework_filename = "PurchasesHybridCommon.framework"
+  build_app(
+    workspace: "ios/PurchasesHybridCommon/PurchasesHybridCommon.xcworkspace",
+    configuration: "Release",
+    scheme: "PurchasesHybridCommon",
+    silent: true,
+    clean: true,
+    derived_data_path: "derived_data/",
+    output_name: framework_filename,
+    destination: 'generic/platform=iOS'
+  )
+
+  # fastlane gym doesn't fully support .frameworks, so we need to look for the output
+  # in derived data manually. 
```

Comment:
this was the reason we weren't using `fastlane gym` before, but I'd much rather do this than deal with carthage's issues